funnel/templates/labels_form.html.jinja2

```diff
@@ -65,7 +65,7 @@
     {%- endfor %}
     <div class="mui-form form-actions clearfix">
       <div>
-        <button type="submit" name="submit" class="mui-btn mui-btn--raised mui-btn--primary" value="reject" data-cy-submit="save-label">Save label</button>
+        <button type="submit" name="submit" class="mui-btn mui-btn--raised mui-btn--primary" data-cy-submit="save-label">Save label</button>
```

Comment:
The word “save” is moving towards meaning “bookmark” and “postpone” in modern UI. For instance, Amazon’s “Save for later” in the checkout flow.

Here the word should be either “Add” or “Done”. @StephanieBr